src/main/resources/messages.yml

```diff
@@ -0,0 +1,48 @@
+message:
+  exception:
+    fileNotFound: "Such file not found!"
+    cardFileNotReadable: "Can't read data from uploaded file!"
+    learningRegimeNotValid: "Value of Learning Regime is not valid - {0}!"
+    numbersOfCardsNegative: "Number of cards should be greater than 0!"
+    cardFieldsNotEmpty: "All of card fields should be filled!"
+    fileEmpty: "File is empty!"
+    fileWrongFormat: "Invalid format of file!"
+    fileDumpingFailed: "Dumping of file failed!"
+    fileCopyFailed: "Copy of file failed!"
+    deckAlreadyExists: "Such deck already exists!"
+    deckNotFound: "Such deck not found!"
```

Comment:
Maybe will be better in this way:
Such deck does not found
BUT I'M NOT SHURE
(I just compared with line 30)

Reply:
It's English tenses)